src/apps/backend/modules/notification/internals/twilio_service.py

```diff
@@ -27,7 +27,10 @@ def send_sms(params: SendSMSParams) -> None:
             )
 
         except TwilioException as err:
-            raise ServiceError(err)
+            raise ServiceError(
```

Comment:
Our system is facing challenge to deliver OTP to you at the moment, and the team has been notified. We recommend you to come back and try again later